### PR TITLE
Coin-435 - Stellar integration

### PR DESCRIPTION
**Pull request by nukky94**
Waiting for the HSM updates (cannot test yet the creation of an account and the signature of tx)

### CODE REVIEW

File: src/main/scala/co/ledger/wallet/daemon/models/Account.scala [hunk 1]
```diff
@@ -420,15 +499,38 @@ object Account extends Logging {
...
+
+  private lazy val stellarPollingScheduler = new ScheduledThreadPoolTimer(
+    poolSize = 1,
+    threadFactory = new NamedPoolThreadFactory("stellar-scheduler-polling-thread")
+  )
+
+  // For Stellar, there's no websocket api but a http sync is fast,
+  // so we use a scheduler to poll operations.
+  def startStellarPolling(poolName: String, walletName: String, a: core.Account)(implicit ec: ExecutionContext): Unit = {
+    stellarPollingScheduler.schedule(
```
Comment (RomanWlm) [inline src/main/scala/co/ledger/wallet/daemon/models/Account.scala:523]: We are working on NRT feature, which is a global account sync management without websocket.
So having stellar specific scheduled task for polling is not necessary.

File: src/main/scala/co/ledger/wallet/daemon/models/Account.scala [hunk 2]
```diff
@@ -421,16 +491,19 @@ object Account extends Logging {
...
+    if (DaemonConfiguration.realTimeObserverOn && !a.isInstanceOfStellarLikeAccount && !a.isObservingBlockchain) {
+      a.startBlockchainObservation()
+      debug(LogMsgMaker.newInstance(s"Set real time observer on ${a.isObservingBlockchain}").append("account", a).toString())
+    }
   }
 
   def stopRealTimeObserver(a: core.Account): Unit = {
-    debug(LogMsgMaker.newInstance("Stop real time observer").append("account", a).toString())
-    if (a.isObservingBlockchain) a.stopBlockchainObservation()
+    if (!a.isInstanceOfStellarLikeAccount && a.isObservingBlockchain) {
```
[the remark this answers is not in the document]
Comment (RomanWlm) [reply src/main/scala/co/ledger/wallet/daemon/models/Account.scala:501]: Okay got it, is there any jira ticket to track this missing implementation ? If we do not expect to deliver realTimeOservation feature in this PR, let's create a ticket and keep WD behavior as it. Once ticket is created, please consider this comment as resolved.

File: src/main/scala/co/ledger/wallet/daemon/models/Wallet.scala [hunk 3]
```diff
@@ -167,10 +166,7 @@ object Wallet extends Logging {
   }
 
   private def startListenAccount(coreA: core.Account): core.Account = {
-    if (DaemonConfiguration.realTimeObserverOn && !coreA.isObservingBlockchain) {
-      coreA.startBlockchainObservation()
-      debug(LogMsgMaker.newInstance(s"Real time observer on ${coreA.isObservingBlockchain}").append("account", coreA.getIndex).toString())
-    }
+    coreA.startRealTimeObserver()
```
[the remark this answers is not in the document]
Comment (RomanWlm) [reply src/main/scala/co/ledger/wallet/daemon/models/Wallet.scala:169]: 👍

File: src/main/scala/co/ledger/wallet/daemon/models/Account.scala (hunk 2 above)
[the remark this answers is not in the document]
Comment (nukky94) [reply src/main/scala/co/ledger/wallet/daemon/models/Account.scala:501]: Yes, https://ledgerhq.atlassian.net/jira/software/projects/COIN/boards/157?selectedIssue=COIN-338

**Pull request merged**